plugins/bufbuild/validate-cpp/v1.1.0/buf.plugin.yaml

```
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/bufbuild/validate-cpp
+plugin_version: v1.1.0
+source_url: https://github.com/bufbuild/protoc-gen-validate
+description: Generates C++ code to validate Protobuf messages using protoc-gen-validate constraints.
+deps:
+  - plugin: buf.build/protocolbuffers/cpp:v25.2
```

```suggestion
  - plugin: buf.build/protocolbuffers/cpp:v27.3
```

Unsure if this is compatible with the latest C++ version.

---

Upstream PGV build is building against older protoc - going to stay on this version as there have been breaking changes in 26.x line.